gen-api-reference/README.md

```diff
@@ -0,0 +1,34 @@
+This folder contains scripts that automatically generate API reference pages for Kubeflow.
+The tool used here can be found at https://github.com/ahmetb/gen-crd-api-reference-docs.
+
+## Prerequisites
+* Clone this repository
+* Clone of the repository for which you are generating reference (e.g. kubeflow/tf-operator).
+
+## Installation
+
+1. Download the tool from https://github.com/ahmetb/gen-crd-api-reference-docs/releases.
+
+1. Extract the tool to a local directory, for example:
+```
+tar -xvf gen-crd-api-reference-docs_linux_amd64.tar.gz -C gen-crd-api-reference-docs
+```
+
+## Usage
+
+1. Open up `gen-tfjob-api.sh` script.
+
+1. Set the `GEN_DOCS` variable to where you have gen-crd-api-reference extracted.
```

Comment:
Should this:
`gen-crd-api-reference`
be this:
`gen-crd-api-reference-docs`
?